app/controllers/profile_controller.rb

```diff
@@ -3,5 +3,7 @@ class ProfileController < ApplicationController
   layout 'card_wide'
 
   def index
+    cacher = Pii::Cacher.new(current_user, user_session)
+    @active_profile = cacher.fetch
```

Comment:
Above, in `saml_idp_auth_concern`, you are doing the same thing, but the method is called `decrypted_pii`, whereas here you are saying the result represents an active profile. IIUC `cacher.fetch` returns a hash of attributes, so it seems like `decrypted_pii` is the correct term, and perhaps the instance variable here should be called that as well?